node/resolvers/session/index.ts

```diff
@@ -38,27 +39,41 @@ const impersonateData = email => {
   }
 }
 
-// Set session cookies to request GET session
-const formatRequestCookie = (cookies) => {
+/** 
+ * Parse session set-cookie array to cookie request format. 
+ * @param cookies
+ * @return parsedCookies
+*/
+export const parseResponseCookies = resCookies => {
   const sessionKey = ['vtex_session', 'vtex_segment']
-  const formatCookie = key => `;${key}=${map(prop(key), filter(path([key]), map(parse, prop('set-cookie', cookies)))).toString()}`
+  const formatCookie = key => `;${key}=${map(prop(key), filter(path([key]), map(parse, prop('set-cookie', resCookies)))).toString()}`
   return sessionKey.reduce((ac, element) => ac + formatCookie(element), '')
 }
 
 export const mutations = {
+  /**
+   * Initialize user session and set vtex_session and vtex_segment cookies
+   * @return Session
+   */
   initializeSession: async (_, args, config) => {
     const { headers } = await makeRequest(_, args, config, paths.session, '{}', 'POST')
-    config.headers.cookie = concat(config.headers.cookie, formatRequestCookie(headers))
+    config.headers.cookie = concat(config.headers.cookie, parseResponseCookies(headers))
 
     const session = await makeRequest(_, args, config, paths.getSession)
 
     config.response.set('Set-Cookie', prop('set-cookie', headers))
     return sessionFields(session.data)
   },
 
+  /**
+   * Impersonate a customer and set clientProfileData in OrderForm
+   * @param email
```

Comment:
Those params aren't the function params but the mutation input. You could say that the parm args received from the GraphQL input are those you've written.